app/resources/forms.js.template

```diff
@@ -369,16 +369,19 @@ function make_form_expandable(expandedSection) {
   });
 }
 
+// Check validation of fields before asking comfirmation
```

Comment:
Optional: It's a bit redundant to say "ask confirmation" twice here. Maybe you can combine the two sentences by "Check validation of fields and ask for ..."